src/Symfony/Component/HttpKernel/DataCollector/EventDataCollector.php

```diff
@@ -52,9 +53,10 @@ public function reset()
 
     public function lateCollect()
     {
-        if ($this->dispatcher instanceof TraceableEventDispatcherInterface) {
+        if ($this->dispatcher instanceof TraceableEventDispatcher) {
```

Comment:
wanted? What about an additional instanceof check before calling `setOrphanedEvents`.

Reply:
Good question.
I went this way because `TraceableEventDispatcherInterface` gets depracted anyway.

@nicolas-grekas, what do you think?

Reply:
Legacy code should continue to work